### PR TITLE
10.5.0

### PR DESCRIPTION
**Pull request by danrodo**
## Description

- Adds Pharmacy search module to assessments
- Removed UDF dependency

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have added tests to cover my changes.
- [ ] No tests are required for this change.
- [ ] My change requires a documentation change.
- [ ] My change requires a CHANGELOG update.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # CirrusMD iOS SDK Changelog
 
+# 10.5.0
+
+## Built with: Xcode 14.2.0
+
+### Changes:
+- Adds Pharmacy search module to assessments
+- Removed UDF dependency
```
Comment (DanielGalarza) [inline CHANGELOG.md:9]: Any other dependency updates or bug fixes worth calling out here? @danrodo @tklucher

**Review by DanielGalarza — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+
+### Bug Fixes:
+
+# 10.4.0
+
```
Comment (tklucher) [inline CHANGELOG.md]: Remove duplicate 10.4.0, this is probably there because we added a 10.4.1

File: CHANGELOG.md (hunk 1 above)
Comment (tklucher) [reply CHANGELOG.md:9]: No, this is the only dependency change for 10.5 all other changes will be in the next release.

**Review by tklucher — CHANGES_REQUESTED** [review]
Looks great only thing is that the CHANGELOG has 10.4.0 twice.

**Review by tklucher — APPROVED** [review]
Looks great!

**Pull request merged**